docs/getting-started/integration.md

```diff
@@ -22,7 +22,21 @@ var myChart = new Chart(ctx, {...});
 
 Chart.js 3 is tree-shakeable, so it is necessary to import and register the controllers, elements, scales and plugins you are going to use.
 
-For all available imports see the example below.
+Until you are working on bundle optimization, the following snippet will ensure all features are available:
+
+```javascript
+import Chart from 'chart.js/auto';
+var myChart = new Chart(ctx, {...});
+```
+
+You can achieve the same thing while explicitly calling `register()` as follows:
+
+```javascript
+import { Chart, registerables } from 'chart.js';
+Chart.register(...registerables);
+```
+
```

Comment:
I'm not sure why anyone would ever need to do this. We don't need to document something that's equivalent to "auto"
```suggestion
```